File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;

    do
    {
        //ask user to enter a height and save it as an integer
        height = get_int("Enter a height\n");

    }
    while (height < 1 || height > 8);


    // Print the pyramid if conditions are met.
    // To achive the left pyramid, the number of # is the same as c, which is the row number.
    // To allign it to the left side, we substract the number of # from the height and print spaces instead.
    for (int c = 1; c <= height; c++)
    {

        for (int r = 1; r <= height; r++)
        {

            if (r <= height - c)
            {

                printf(" ");

            }

            else
            {

                printf("#");

            }

        }

        //Print the space between both pyramid
        printf("  ");

        //Print the right pyramid
        for (int r = 1; r <= c; r++)
        {

            printf("#");

        }

        printf("\n");



    }

}

File: pset6/mario/more/mario.py
from cs50 import get_int

height = 0

# Ask user to enter the height of desired pyramid
while height < 1 or height > 8:
    height = get_int("Height: ")

# Print left pyramid
for c in range(height):
    for r in range(height):
        if (r < height - c - 1):
            print(" ", end="")
        else:
            print("#", end="")
            
    # Print space between pyramid        
    print("  ", end="")
    
    # Print right pyramid
    for r in range(c + 1):
        print("#", end="")
    print()